lib/translators.js

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import {
+  ECDSA_2019_SECP_256_KEY_TYPE,
+  ECDSA_2019_SECP_384_KEY_TYPE,
+  ECDSA_2019_SECP_521_KEY_TYPE,
+  ECDSA_2019_SUITE_CONTEXT_V1_URL,
+  MULTIKEY_CONTEXT_V1_URL
+} from './constants.js';
+
+// converts key pair to Multikey format
+export async function toMultikey({keyPair}) {
+  const validEcdsaTypes = [
```

Comment:
Move out of the function and into a constant in the file. The values in the array are constant between function invocations, therefore we don't need to create a new array on each function invocation.

Also, change the array to a set after moving the code.